### PR TITLE
360-ds: instructions and generate version key automatically

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by odscjames** [comment]
[360-ds: instructions and generate version key automatically](https://trello.com/c/REzZeZGx/224-360-ds-instructions-and-generate-version-key-automatically)

**Comment by odscjames** [comment]
Generates URL's like: /static/css/main.css?v=1637245359.1435149

I think this will be deploy time not source file commit time - so technically we leak a bit of information about when we deploy. But I'm pretty sure we don't care about that.

**Review by michaelwood — CHANGES_REQUESTED** [review]
Thanks, great to have that css stuff documented. 

All looks good to me. That minor comment and I do think this is 3 commits really though 1) add docs 2) add context processor 3) update to use context processor

Related issue - https://github.com/ThreeSixtyGiving/grantnav/issues/666

**Comment by odscjames** [comment]
> I do think this is 3 commits really though 1) add docs 2) add context processor 3) update to use context processor

I would have said 2 - 2 & 3 really go together. I'll do that and add a comment.

File: grantnav/frontend/context_processors.py [hunk 1]
```diff
@@ -13,3 +16,10 @@ def navigation(request):
         {"name": "Get Help", "url": "https://help.grantnav.threesixtygiving.org/", "id": "help_link"},
     ]
     return {'nav_menu': sections}
+
+
+MAIN_CSS_CACHE_KEY = getmtime(os.path.join(settings.BASE_DIR, 'grantnav', 'frontend', 'static', 'css', 'main.css'))
```
[the remark this answers is not in the document]
Comment (odscjames) [reply grantnav/frontend/context_processors.py:21]: Yes

**Review by michaelwood — APPROVED** [review]

**Pull request merged**